[stable/prometheus] Add seccomp/apparmor annotations on node-exporter PSP

spiarh opened the pull request:
…tyPolicy

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

<!--
Thank you for contributing to helm/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed, a CircleCI
will run across your changes and do some initial checks and linting. These checks run
very quickly. Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

#### What this PR does / why we need it:
Add these annotation to the PSP, these annotations can still be "overridden" with pods annotations.
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
    seccomp.security.alpha.kubernetes.io/defaultProfileName: 'docker/default'
    apparmor.security.beta.kubernetes.io/defaultProfileName: 'runtime/default'

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

Commit:
[stable/prometheus] add seccomp and apparmor annotations on PodSecurityPolicy

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
[stable/prometheus] add seccomp and apparmor annotations on PodSecurityPolicy

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

patsevanton committed:
Fix data.persistence.storageClass in example in incubator/elasticsearch/README.md (#8503)

* Fix data.persistence.storageClass in example

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

* Revert "Fix data.persistence.storageClass in example"

This reverts commit 8de74be89c4de9bf852d9d50078066555c12595f.

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

* Fix data.persistence.storageClass in example in incubator/elasticsearch/README.md
Signed-off-by: Anton Patsev patsev.anton@gmail.com

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

* Fix data.persistence.storageClass in example in README.md
Signed-off-by: Anton Patsev patsev.anton@gmail.com

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

ehardon committed:
Allow adding custom ldif files to the openldap chart (#7396)

* Update deployment.yaml

Updated deployment.yaml file in order to provide the desired argument `--copy-service` to start the ldap docker image so that a user of the chart can override the basis ldif file with another provided as an argument when installing the chart.

Signed-off-by: ehardon <elena.hardon@ness.com>

* Added precondition for applying copy argument

Added precondition for the `--copy-service` argument so that it won't be applied if no custom ldif files are provided.

Signed-off-by: Elena Hardon <elena.hardon@ness.com>
Signed-off-by: ehardon <elena.hardon@ness.com>

* Updated chart version

Signed-off-by: Elena Hardon elena.hardon@ness.com
Signed-off-by: ehardon <elena.hardon@ness.com>

* bump version

Signed-off-by: David J. M. Karlsen <david@davidkarlsen.com>

carrodher committed:
[stable/phabricator] Remove distro tag (#8542)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/phpbb] Remove distro tag (#8537)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/phpmyadmin] Remove distro tag (#8538)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/owncloud] Remove distro tag (#8540)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/parse] Remove distro tag (#8541)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/suitecrm] Remove distro tag (#8531)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/testlink] Remove distro tag (#8532)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/wordpress] Remove distro tag (#8533)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/rabbitmq] Remove distro tag (#8534)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/redis] Remove distro tag (#8535)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/moodle] Remove distro tag (#8530)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/prestashop] Remove distro tag (#8539)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/opencart] Remove distro tag (#8543)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/orangehrm] Remove distro tag (#8544)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/osclass] Remove distro tag (#8545)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/mongodb] Remove distro tag (#8546)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/odoo] Remove distro tag (#8548)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/magento] Remove distro tag (#8549)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/mariadb] Remove distro tag (#8550)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/mediawiki] Remove distro tag (#8551)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/jasperreports] Remove distro tag (#8552)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/joomla] Remove distro tag (#8553)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/dokuwiki] Remove distro tag (#8554)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/drupal] Remove distro tag (#8555)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/ghost] Remove distro tag (#8556)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/nats] Remove distro tag (#8547)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

balboah committed:
gocd: wrong agent resource value ref (#8496)

* gocd: wrong agent resource value ref

.Values.agent.resources exist but is not used

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

* gocd: Update changelog

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

Commit:
[stable/phabricator] Release 3.2.2 (#8557)

* phabricator: update to `2018.41.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* phabricator: bump chart appVersion to `2018.41.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* phabricator: bump chart version to `3.2.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

cpanato committed:
Bump version to 5.4.0 (#8522)

Signed-off-by: cpanato <ctadeu@gmail.com>

carrodher committed:
[stable/redmine] Remove distro tag (#8536)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

Commit:
[stable/orangehrm] Release 3.1.2 (#8566)

* orangehrm: update to `4.1.2-1`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* orangehrm: updated chart requirements

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* orangehrm: bump chart appVersion to `4.1.2-1`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* orangehrm: bump chart version to `3.1.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

RTodorov committed:
Spinnaker/add azs support (#8564)

* Add support to Azure Storage

Signed-off-by: Renato Todorov <renato.todorov@gmail.com>

* Add supported storage types to README file. Bump chart version.

Signed-off-by: Renato Todorov <renato.todorov@gmail.com>

desaintmartin committed:
[stable/kibana] Fix empty parameters to have valid Kubernetes and Helm YAML files. (#8284)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

a-robinson committed:
[stable/cockroachdb] Warn about default disk size, improve NOTES format (#7828)

The default disk size of 1 GiB can cause mistakes (e.g.
https://stackoverflow.com/q/52245811/1925481), but is needed in
order to work in environments like Minikube.

And I don't know what's going on with the way that line breaks work in
NOTES.txt, but this is what makes it render properly.

Signed-off-by: Alex Robinson <alexdwanerobinson@gmail.com>

jar361 committed:
Update elastic-stack to using fluentd version 1.0.0 (#8585)

Update elastic-stack to using logstash version 1.0.0

Signed-off-by: Johnathan Rodgers <jrodgers@powerhrg.com>

Commit:
[stable/parse] Release 3.1.2 (#8584)

* parse: update to `3.1.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* parse: bump chart appVersion to `3.1.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* parse: bump chart version to `3.1.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/drupal] Release 2.2.2 (#8575)

* drupal: update to `8.6.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* drupal: updated chart requirements

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* drupal: bump chart appVersion to `8.6.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* drupal: bump chart version to `2.2.2`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

Commit:
[stable/ghost] Release 5.3.4 (#8578)

* ghost: update to `2.2.4`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* ghost: updated chart requirements

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* ghost: bump chart appVersion to `2.2.4`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* ghost: bump chart version to `5.3.4`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

cpanato committed:
bump weave cloud (#8572)

Signed-off-by: cpanato <ctadeu@gmail.com>

alexandrst88 committed:
Added Pod annotations for Nexus Helm Chart (#8569)

Signed-off-by: Oleksandr Stepanov <alexandrst88@gmail.com>

ian28223 committed:
configurable container name (#8592)

* sets container name as defined in values file

use container name defined in Values file else use Chart name by default

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

* sets container name as defined in values file

use container name defined in Values file else use Chart name by default

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

* changes default to chart name

prior to this PR, the container name had been "datadog" (Chart name). suddenly changing to dd-agent might/could break things for those upgrading.

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

* makes cluster agent container name configurable

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

* adds settings to the readme

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

* bump version

Signed-off-by: Marc Ian Bucad <ian.bucad@gmail.com>

desaintmartin committed:
[stable/kibana] use checksum/config annotation to restart pod if configuration changed. (#8374)

* [stable/kibana] use checksum/config annotation to restart pod if configuration changed.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* [stable/kibana] Outdent if.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

rendhalver committed:
Promote elasticsearch-curator to stable (#8613)

* Promote elasticsearch-curator to stable

Signed-off-by: Pete Brown <pete.brown@powerhrg.com>

* Fix references to incubator/elasticsearch-curator

Signed-off-by: Pete Brown <pete.brown@powerhrg.com>

zdw committed:
stable/kibana: Fix JSON test, clarify log messages (#8517)

Add test values for dashboards to ci directory

Signed-off-by: Zack Williams <zdw@opennetworking.org>

desaintmartin committed:
[stable/horovod] Major version bump: Fix chart not being upgradable. (#8596)

See https://github.com/helm/charts/issues/7680.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

okgolove committed:
[stable/drone], changed repository name in NOTES  (#8628)

* Changed repository name in NOTES

Signed-off-by: Mikhail Naletov <okgolove@markeloff.net>

* Bumped chart version

Signed-off-by: Mikhail Naletov <okgolove@markeloff.net>

legal90 committed:
[stable/kubernetes-dashboard] Use the target port number instead of a name (#7548)

* [stable/kubernetes-dashboard] Use the target port number instead of a name

Older (but still actual) versions of some Network Policy providers don't support named ports as
a value of `targetPort` field in the Service spec.
Changing it to the number shouldn't break the UX for existing installations.

Signed-off-by: Mikhail Zholobov <legal90@gmail.com>

* [stable/kubernetes-dashboard] Bump chart version to 0.7.4

Signed-off-by: Mikhail Zholobov <legal90@gmail.com>

yurrriq committed:
[stable/redis] Avoid generating null annotations (#8588)

Signed-off-by: Eric Bailey <e.bailey@sportradar.com>

nestorsalceda committed:
Download container images from falcosecurity organization (#8560)

Signed-off-by: Néstor Salceda <nestor.salceda@sysdig.com>

fabriziofortino committed:
[incubator/elasticsearch-curator] include annotations only when non e… (#7898)

* [incubator/elasticsearch-curator] include annotations only when non empty

Signed-off-by: Fabrizio Fortino <fabrizio.fortino@gmail.com>

* [incubator/elasticsearch-curator] bump chart version

Signed-off-by: Fabrizio Fortino <fabrizio.fortino@gmail.com>

* es-curator: bump version

Signed-off-by: Fabrizio Fortino <fabrizio.fortino@gmail.com>

* es-curator: fix for #7898 in stable

Signed-off-by: Fabrizio Fortino <fabrizio.fortino@gmail.com>

desaintmartin committed:
[stable/fluentd-elasticsearch] Add nodeSelector support. (#8367)

Also fix values in readme.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

Jonher937 committed:
[stable/unifi] Invalid link in chart sources (#8639)

* Invalid link in chart sources

There was a invalid link in sources, this commit fixes the link
Signed-off-by: Jonathan Herlin <jonte@jherlin.se>

* stable/unifi bump version

Signed-off-by: Jonathan Herlin <jonte@jherlin.se>

Btodhunter committed:
add brady todhunter as approver to anchore-engine (#8614)

Signed-off-by: Brady Todhunter <bradyt@anchore.com>

balboah committed:
Gocd/fix affinity (#8521)

* gocd: Invalid nodeSelector reference

Missing "agent" in if case

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

* gocd: Add affinity setting

Agent and server affinity values for more flexibility.
Node selector will be deprecated in future k8s

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

* gocd: Bump version and changelog

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

* gocd: empty line messing up README

Signed-off-by: Johnny Bergström <johnny@klaudify.se>

arvindsv committed:
[stable/gocd] Update readme - Add link to getting started (#8648)

* [stable/gocd] Add intro guide link in README

Signed-off-by: Aravind SV <mail@arvindsv.com>

* [stable/gocd] Bump version and update changelog

Signed-off-by: Aravind SV <mail@arvindsv.com>

dcarley committed:
[stable/mongodb-replicaset] Add OU to cert (#8261)

* [stable/mongodb-replicaset] Add OU to cert

Fix the bug reported and diagnosed in #7417.

When you set `security.clusterAuthMode: x509` as recommended by the
README the nodes will fail to authenticate with the following error:

    2018-10-08T13:06:12.767+0000 I ACCESS   [conn7]  authenticate db: $external { authenticate: 1, mechanism: "MONGODB-X509", user: "CN=circleci-mongodb-1" }
    2018-10-08T13:06:12.767+0000 I ACCESS   [conn7] Failed to authenticate CN=circleci-mongodb-1@$external with mechanism MONGODB-X509: UserNotFound: Could not find user CN=circleci-mongodb-1@$external
    2018-10-08T13:06:12.768+0000 I ACCESS   [conn7] Unauthorized: not authorized on admin to execute command { replSetHeartbeat: "rs0", configVersion: 3, from: "circleci-mongodb-1.circleci-mongodb.default.svc.cluster.local:27017", fromId: 1, term: 12 }

This is because the MongoDB docs state the following:

> The Distinguished Name (DN), found in the member certificate’s
> subject, must specify a non-empty value for at least one of the
> following attributes: Organization (O), the Organizational Unit (OU)
> or the Domain Component (DC).

https://docs.mongodb.com/manual/tutorial/configure-x509-member-authentication/#certificate-requirements

Setting the OU to the name of the product seems like the most generic
solution to this without suggesting that the cert belongs to MongoDB the
company.

Signed-off-by: Dan Carley <dan.carley@gmail.com>

* [stable/mongodb-replicaset] Fix CAFile in README

The `/ca` directory doesn't exist when the `bootstrap` container is run
and this causes the `on-start.sh` script to fail. Whereas the
`/data/configdb` directory does existing throughout the process and is
the path that people have reported using in other issues.

Signed-off-by: Dan Carley <dan.carley@gmail.com>

Commit:
[stable/ghost] Release 5.3.5 (#8644)

* ghost: update to `2.3.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* ghost: bump chart appVersion to `2.3.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* ghost: bump chart version to `5.3.5`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

joaocc committed:
[stable/mysql] Fix secret naming convention (#7883)

* Fix secret naming convention

I believe https://github.com/helm/charts/pull/5783 missed this secret name update 
(it is using mysql.fullname instead of mysql.secretname)

Signed-off-by: Joao C Costa <joaocc-dev@live.com>

* Bump version

Signed-off-by: Joao C Costa <joaocc-dev@live.com>

irlevesque committed:
Enables specifying Priority Class for Datadog Agent pods (#8642)

Signed-off-by: Ian Levesque <ian@quantopian.com>

javsalgar committed:
[stable/nats] Fix cluster auth random password (#8652)

Signed-off-by: Javier J. Salmeron Garcia <jsalmeron@bitnami.com>

gianrubio commented on stable/prometheus/templates/node-exporter-podsecuritypolicy.yaml:
```diff
@@ -10,6 +10,10 @@ metadata:
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: 'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName: 'runtime/default'
```
shall we make this optional?

spiarh replied:
I think it is a good practice to keep it by default but making this optional might be safer in case apparmor is not enabled.

FlavioF committed:
[stable/prometheus] Add extra volume directories to configmapreload (#8601)

Signed-off-by: Flávio Ferreira <flaviommferreira@gmail.com>

vvinnich committed:
[stable/keycloak] Fix typo Readme.md Custom values for Google Cloud SQL (#8224)

* [stable/keycloak] Fix typo Readme.md Custom values for Google Cloud SQL

Signed-off-by: Vitalii Vinnichuk <vinnichukvitalii@gmail.com>

* [stable/keycloak] bump version

Signed-off-by: Vitalii Vinnichuk <vinnichukvitalii@gmail.com>

* Update README.md

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

Commit:
updating version of the Datadog Cluster Agent (#8658)

Signed-off-by: charlyF <charly@datadoghq.com>

Commit:
Added ExtraPorts to open in the master pod (#7759)

* Added ExtraPorts to open in the master pod

Signed-off-by: ecejjar <jesus.javier.arauz@ericsson.com>

* [stable/jenkins] Bumped up chart version

Signed-off-by: ecejjar <jesus.javier.arauz@ericsson.com>

Towmeykaw committed:
[stable/fluent-bit] Added ability to specify extra Input, Filter and Output sections (#8600)

* Added ability to specify extra Input, Filter and Output sections

Signed-off-by: towmeykaw <tkindmark@gmail.com>

* Updated README

Signed-off-by: towmeykaw <tkindmark@gmail.com>

* Bumped chart version

Signed-off-by: towmeykaw <tkindmark@gmail.com>

* Removed trailing space

Signed-off-by: towmeykaw <tkindmark@gmail.com>

cpanato committed:
bump version (#8574)

Signed-off-by: cpanato <ctadeu@gmail.com>

gianrubio committed:
Migrate prometheus operator from coreos (#6765)

* Create promethues-operator chart

Signed-off-by: Giancarlo Rubio <gianrubio@gmail.com>

* Fixes for stable/prometheus-operator

Changed:
- Update Prometheus to 2.4.3, alertmanager to 0.15.2
- Change some job labels - should refer to a label appearing on the service. Tried to keep this consistent with what was used in the coreos version
- Add confguration for using CoreDNS
- Add insecureSkipVerify to kubeApiServer
- Add `prometheus.additionalScrapeConfigs`, `prometheus.additionalAlertManagerConfigs`

- Add ability to deploy an operated prometheus without installing anything else
  - Switch `prometheus.serviceAccountName` to be based on `prometheus.fullname` to prevent clashes
- Fix `serviceMonitorSelector` typo
- Add `serviceMonitorNamespaceSelector` to values.yaml
- Fix grafana typo (https://github.com/helm/charts/pull/6765#issuecomment-428653560)

Add CoreDNS dashboard

- Fixed dashboard use of updated name metric

Signed-off-by: Giancarlo Rubio <gianrubio@gmail.com>
Signed-off-by: Vasily Sliouniaev <vsliouniaev@users.noreply.github.com>

Vas prometheus operator (#5)

* Fix lint

* Fix missing selector on kubelet

https://github.com/helm/charts/pull/6765#issuecomment-431035212

* Fix braces in files

* Fix naming

* Move non-exporters out of exporters folder

* Use common value names, add to readme

* Change value names deploy -> enabled

https://github.com/helm/charts/pull/6765#issuecomment-428283768
https://github.com/helm/charts/pull/6765#issuecomment-430070684

* Update etcd rules

https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/etcd3_alert.rules.yml

* Fix etcd rules to match job name

* Review readme

* Change initial version to 0.1.0

Additional Fixes While Github is down (#6)

* Fix additional service monitor template

* Fix label indentation

https://github.com/helm/charts/pull/6765#issuecomment-431788727

Fix cleanup, requirements.lock file (#7)

solidnerd committed:
Update traefik to 1.7.3 (#8615)

Signed-off-by: solidnerd <niclas@mietz.io>

patsevanton committed:
Up version sorintlab/stolon to v0.12.0-pg10, etcd-amd64 to 2.3.7 (#8625)

* Up version sorintlab/stolon to v0.12.0-pg10, etcd-amd64 to 2.3.7

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

* Up version in Stolon version: 0.4.3, appVersion: 0.12.0

Signed-off-by: Anton Patsev <patsev.anton@gmail.com>

mgrachev committed:
[stable/express-gateway] fix typo (#8632)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

desaintmartin committed:
[incubator/patroni] Fix chart not being upgradable because of spec.VolumeClaimTemplate.metadata.labels being immutable (#8594)

See https://github.com/helm/charts/issues/7803

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

Commit:
[stable/phabricator] Release 3.2.3 (#8654)

* phabricator: update to `2018.42.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* phabricator: updated chart requirements

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* phabricator: bump chart appVersion to `2018.42.0`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

* phabricator: bump chart version to `3.2.3`

Signed-off-by: Bitnami Containers <containers@bitnami.com>

guptaarvindk committed:
Mongodb/ipv6 (#7674)

* suport for:
- headless services and static clusterIP
- option to disable the IPV6 even in standalone deploymnet.

Signed-off-by: Arvind Gupta <guptaarvindk@gmail.com>

* documentation for previous commit (clusterIP, headless service, ipv6 disable)

Signed-off-by: Arvind Gupta <guptaarvindk@gmail.com>

* bumped chart version.
fixed typo in values.yaml

Signed-off-by: Arvind Gupta <guptaarvindk@gmail.com>

yuchaoran2011 committed:
Specified webhook service namespace, updated default image, and added support for existing namespace (#7874)

* Specify webhook service namespace

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Bumped version number

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Updated default image

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Support installing operator in an existing namespace. Support Spark RBAC setup in an existing namespace

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

* Added OWNERS

Signed-off-by: Chaoran Yu <yuchaoran2011@gmail.com>

andig committed:
Correct repo name and ingress labels (#8662)

Signed-off-by: andig <cpuidle@gmx.de>

hbagdi committed:
Kong chart: enable persistence by default for postgres (#8480)

* Kong chart: enable persistence by default for postgres

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

* Update Chart.yaml

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

vsliouniaev committed:
Correct prometheus-operator app version (#8665)

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

monotek committed:
fixed annotations (#8431)

Signed-off-by: André Bauer <andre.bauer@kiwigrid.com>

nhuray committed:
[stable/hlf-ca] Support MySQL, Postgres and External Database (#8610)

* Update README.md to declare persistence.existingClaim value

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Define helm dependency to install postgres or mysql

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Check the database type at container startup

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Configure mysql to work with the Fabric CA schema

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Add template in _helpers.tpl

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Handle the case the port is not passed when setup an external DB

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Update the README.md

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Fix linting errors

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Define an initContainer to wait for database availability

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Add missing liveness and readiness probes

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Update helm dependency

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

* Update README.md

Signed-off-by: Nicolas Huray <nhuray@estateably.com>

Commit:
[stable/redis-ha] Update image to quay.io/smile/redis:4.0.11-r1 (#8650)

Signed-off-by: Franz Luther Neulist Carroll <franzneulistcarroll@gmail.com>

carrodher committed:
[stable/postgresql] PostgreSQL maintenance proposal (#8004)

* Adopt postgresql chart

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Fix Chart.yaml

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add metrics and NetworkPolicy to the README

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Remove previous deployment.yaml

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Update NOTES.txt

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add OWNERS file

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add kubeapps text to charts READMEs

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Appy different suggestions

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add terminationGracePeriodSeconds

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add upgrade steps to README

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add how to connect when the networkpolicy is enabled to NOTES.txt

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Allow using pg_hba.conf via configmap

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Update description ingluding pg_hba.conf

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Fix metrics deployment

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Rebase latest changes

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Remove distro tags

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Add OWNERs to .helmignore

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

tyranron committed:
[stable/nginx-ingress] Fix incorrect YAML indentation (#8303)

Signed-off-by: tyranron <tyranron@gmail.com>

hairyhenderson committed:
[prometheus-blackbox-exporter] Allowing custom service annotations and labels (#8673)

Signed-off-by: Dave Henderson <dhenderson@gmail.com>

vsliouniaev committed:
Update readme with helm workaround (#8670)

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

carrodher committed:
[stable/postgresql] Add global registry option and remove some quotes (#8684)

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

mgrachev committed:
[stable/hl-composer] fix typo (#8679)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/hlf-ca] fix typo (#8680)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

desaintmartin committed:
[stable/redis] Add myself as maintainer. (#8682)

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

carrodher committed:
[stable/redmine] Update PostgreSQL as deps (#8686)

* [stable/redmine] Update PostgreSQL as deps

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Fix url

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Bump major version

Co-Authored-By: carrodher <carrodher1179@gmail.com>
Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

desaintmartin committed:
[stable/rabbitmq] Add myself as maintainer. (#8688)

* [stable/rabbitmq] Add myself as maintainer.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Add OWNERS to helm ignore list.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

kirrmann committed:
[stable/prometheus] Add cluster.advertise-address to alertmanager cmd args (#8562)

Signed-off-by: Stefan Kirrmann <stefan.kirrmann@gmail.com>

carrodher committed:
[stable/redmine] Fix postgresql password secret (#8690)

* Fix password secret

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* [stable/redmine] Fix postgresql password secret

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

carrodher committed:
[stable/odoo] Update PostgreSQL as deps (#8687)

* [stable/odoo] Update PostgreSQL as deps

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Fix url

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Bump major version

Co-Authored-By: carrodher <carrodher1179@gmail.com>
Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

* Fix password secret

Signed-off-by: Carlos Rodriguez Hernandez <crhernandez@bitnami.com>

hkaj committed:
Extend container runtime support to CRI (#8661)

Signed-off-by: Haissam Kaj <haissam.kaj@gmail.com>

spiarh committed:
[stable/elasticsearch] Add ServiceAccounts for each components + PodSecurityPolicy (#8529)

* [stable/elasticsearch] Add ServiceAccounts for each components

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

* Add PodSecurityPolicy support

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

* minor fixes

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
[stable/postgreslq] Improve service configuration (#8691)

* Fix typo

Signed-off-by: juan131 <juan@bitnami.com>

* [stable/postgreslq] Improve service configuration

Signed-off-by: juan131 <juan@bitnami.com>

* Fix lint issues

Signed-off-by: juan131 <juan@bitnami.com>

* Small corrections on README.md file

Signed-off-by: juan131 <juan@bitnami.com>

* Small corrections on README.md file - ii

Signed-off-by: juan131 <juan@bitnami.com>

mgrachev committed:
[stable/hlf-ord] fix typo (#8695)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/hlf-peer] fix typo (#8696)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

mgrachev committed:
[stable/home-assistant] fix typo (#8702)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

hairyhenderson committed:
[prometheus-blackbox-exporter] Fixing minor syntax error (#8674)

Signed-off-by: Dave Henderson <dhenderson@gmail.com>

paskal committed:
fix typos in prometheus-operator (#8705)

* fix typos

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

* bump version

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

* fix typo

Signed-off-by: Dmitry Verkhoturov <paskal.07@gmail.com>

irlevesque committed:
[stable/datadog] Add support for mapping Kubernetes Labels and Annotations to Datadog Tags (#8643)

* Add support for mapping Kubernetes Labels and Annotations to Datadog
Tags

Signed-off-by: Ian Levesque <ian@quantopian.com>

* use YAML representation of mappings

Signed-off-by: Ian Levesque <ian@quantopian.com>

* Update default values examples

Signed-off-by: Ian Levesque <ian@quantopian.com>

mgrachev committed:
[stable/kuberos] fix typo (#8704)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

ljfranklin committed:
livenessProbe in kube-dashboard is configurable (#8621)

- Adds configurable `livenessProbe.initialDelaySeconds` and
  `livenessProbe.timeoutSeconds`

Signed-off-by: Lyle Franklin <lylejfranklin@gmail.com>

mgrachev committed:
[stable/mariadb] fix typo (#8701)

Signed-off-by: Mikhail Grachev <work@mgrachev.com>

cpanato committed:
bump minio and mc (#8570)

Signed-off-by: cpanato <ctadeu@gmail.com>

desaintmartin committed:
Guidelines: set matchLabels as being mandatory (#7692)

* Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* fixup! Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* fixup! fixup! Guidelines: set matchLabels as being mandatory.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Update guidelines: mention DaemonSets as well.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: be more precise + specify upgrade

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: fix typos, add persistence paragraph and do not repeat component part.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Review guidelines: add PVC paragraph.

Signed-off-by: Cédric de Saint Martin <cdesaintmartin@wiremind.fr>

* Fix spelling/typos

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

* Fix incorrect typo fix

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

Commit:
7552 | Change path of health checks and add init delay (#7554)

* 7552 | Change path of health checks and add init delay

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Incrementing chart version

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Adding configuration for readinessDelay

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Updating chart version to master

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

* 7552 | Bumping chart version.

Signed-off-by: Matthew Greenwald <mgreenwald@gmail.com>

vsliouniaev committed:
Update readme and examples (#8709)

Signed-off-by: Vasily Sliouniaev <vasily.sliouniaev@jet.com>

cablespaghetti committed:
Upgrade to Kafka 2.0.0 and disable confluent metric submission (#8693)

Signed-off-by: Sam Weston <weston.sam@gmail.com>

Commit:
[stable/prisma]: add service account (#8716)

Signed-off-by: Giacomo Guiulfo <giacomoguiulfo@gmail.com>

adoy committed:
[Minio] Fix domain and browser config (#8683)

Following https://github.com/minio/minio/pull/6498/files
domain and config are deprecated in config entries. Use env variable instead

Signed-off-by: Pierrick Charron <pierrick@webstart.fr>

Commit:
Make pod annotations configurable for PodSecurityPolicy

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
Make pod annotations configurable for PodSecurityPolicy

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
bump chart version

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
bump chart version

Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

Commit:
Merge branch 'add_annotation_node_exporter_psp' of github.com:lcavajani/charts into add_annotation_node_exporter_psp
